src/test/resources/pages/adminEmailCompose.html

```diff
@@ -347,7 +344,7 @@
                                 </span>
                             </form>
     
-                            <div id="documentBaseUrl">${app.url}</div>
+                            {*}
```

Comment:
Not sure about this because I'm not working on it, but any side effect if we keep the previous one?